test/easypost.js

```diff
@@ -126,13 +126,6 @@ describe('Base API object', () => {
       });
     });
 
-    it('requets json', done => {
```

Comment:
What was this? (and why does it say 'requets'?)

Reply:
Someone left a typo 😉 

It was a test asserting we passed `Accept json` into the headers, which we now do elsewhere and so this test is now dead.